frontend/src/components/Profile/TaskDetail.js

```diff
@@ -0,0 +1,176 @@
+import classes from "./TaskDetail.module.css";
+import { useEffect, useState } from "react";
+import { useParams, useHistory } from "react-router-dom";
+
+const TaskDetail = () => {
+  const [taskDetail, setTaskDetail] = useState([]);
+  const BACKEND_API = "http://127.0.0.1:4000";
+  const params = useParams();
```

Comment:
this component receives `params` prop from `TaskDetailPage`, meaning that you don't need to call this hook to retrieve params again. you can read the value from props.